alpha/apps/kaltura/lib/batch2/bcdl/kBusinessConvertDL.php

```diff
@@ -22,6 +22,19 @@ public static function replaceEntry(entry $replacedEntry, entry $replacingEntry
 			return;
 		}
 
+		$lock = kLock::create("replacement_" . $replacedEntry->getId() . "_" . $replacingEntry->getId());
+		if ($lock && !$lock->lock())
```

Comment:
Consider setting the lock timeout higher the default of 2 seconds since the replacement process may be longer